.rubocop.yml

```diff
@@ -1,3 +1,8 @@
+AllCops:
+  Exclude:
+    - 'spec/rails_helper.rb'
```

Comment:
Same as before, if you're lazy you can use [the bootstrap's rubocop.yml](https://github.com/Wolox/rails-bootstrap/blob/master/.rubocop.yml), it should be generic enough for most Rails API projects